content/en/blog/2023/sig-testing-spotlight.md

```diff
@@ -0,0 +1,203 @@
+
+**Patrick:** Hello! I work as a software engineer and architect in a
+team at Intel which focuses on open source Cloud Native projects. When
+I ramped up on Kubernetes to develop a storage driver, my very first
+question was "how do I test it in a cluster and how do I log
+information?" That interest led to various enhancement proposals until
+I had (re)written enough code that also took over official roles as
+SIG Testing Tech Lead (for the [E2E framework](https://github.com/kubernetes-sigs/e2e-framework)) and
+structured logging WG lead.
+
+## Testing practices and tools
+
+**Sandipan:** Testing is a field in which multiple approaches and
+tools exist; how did you arrive at the existing practices?
+
+**Patrick:** I can’t speak about the early days because I wasn’t
+around yet 😆, but looking back at some of the commit history it’s
+pretty obvious that developers just took what was available and
+started using it. For E2E testing, that was
+[Ginkgo+Gomega](https://github.com/onsi/ginkgo). Some hacks were
+necessary, for example around cleanup after a test run and for
+categorising tests. Eventually this led to Ginkgo v2 and [revised best
+practices for E2E testing](https://www.kubernetes.dev/blog/2023/04/12/e2e-testing-best-practices-reloaded/).
+Regarding unit testing opinions are pretty diverse: some maintainers
+prefer to use just the Go standard library with hand-written
+checks. Others use helper packages like stretchr/testify. That
+diversity is okay because unit tests are self-contained - contributors
+just have to be flexible when working on many different areas.
+Integration testing falls somewhere in the middle. It’s based on Go
+unit tests, but needs complex helper packages to bring up an apiserver
+and other components, then runs tests that are more like E2E tests.
+
+## Subprojects owned by SIG Testing
+
+**Sandipan:** SIG Testing is pretty diverse. Can you give a brief
+overview of the various subprojects owned by SIG Testing?
+
+**Michelle:** Broadly, we have subprojects related to testing
+frameworks, and infrastructure, though they definitely overlap.  So
+for the former, there's
+[e2e-framework](https://pkg.go.dev/sigs.k8s.io/e2e-framework) (used
+externally),
+[test/e2e/framework](https://pkg.go.dev/k8s.io/kubernetes/test/e2e/framework)
+(used for Kubernetes itself) and kubetest2 for end-to-end testing,
+as well as boskos (resource rental for e2e tests),
+[KIND](https://kind.sigs.k8s.io/) (Kubernetes-in-Docker, for local
+testing and development), and the cloud provider for KIND.  For the
+latter, there's [Prow](https://docs.prow.k8s.io/) (K8s-based CI/CD and
+chatops), and a litany of other tools and utilities for triage,
+analysis, coverage, Prow/TestGrid config generation, and more in the
+test-infra repo.
+
+*If you are willing to learn more and get involved with any of the SIG
+Testing sub-projects, check out the [SIG Testing README](https://github.com/kubernetes/community/tree/master/sig-testing#subprojects).*
```

Comment:
Thank you @fsmunoz! A nit:
```suggestion
Testing subprojects, check out the [SIG Testing README](https://github.com/kubernetes/community/tree/master/sig-testing#subprojects).*
```

Reply:
Yes, we have both present in the documentation and blogs, but subprojects is likely better (and it's your text to begin with) so I'll add it.